docs/gds-supported-platform/accessing-dashboard.md

```diff
@@ -1,29 +1,13 @@
 # Accessing the Kubernetes Dashboard
 
-Each cluster will have a deployment of the kubernetes dashboard available. This will not have an ingress configured and so will not be accessible from outside the cluster. It will however work via a proxy tunnel. The service account the dashboard is running under has very limited access (making the dashboard useless) so authenticating against the dashboard is also required. This guide will walk through how to connect to the dashboard and authenticate using your IAM credentials.
+## Using the GDS CLI
 
-## Requirements
+1. `gds dashboard sandbox`
+1. Copy and paste the provided token into the dashboard login form
 
-* [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator)
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* a kubeconfig as set up by the cluster terraform
+## Without using the GDS CLI
```

Comment:
Why do we even give this as an option? The GDS CLI should be the standard.

Reply:
I mean I don't use it necessarily—not for a particularly compelling reason but I thought it was worth showing the other way.